README.md

```diff
@@ -39,8 +39,8 @@ Intel Extension for PyTorch is a Python package to extend official PyTorch. It i
  3. Add an new backend for Intel PyTorch Extension
     ```bash
     # Apply git patch to pytorch code
-    cd ${intel_pytorch_extension_directory}
-    git apply torch_patches/dpcpp-v1.5-rc3.patch ${pytorch_directory}
+    cd ${pytorch_directory}
```

Comment:
We should change the work directory to `${intel_pytorch_extension_directory}` but not `${pytorch_directory}`. Because we need to apply the patch and the patch is under `${intel_pytorch_extension_directory}/torch_patches/dpcpp-v1.5-rc3.patch`

Reply:
When I apply intel patch in ${intel_pytorch_extension_directory} to ${pytorch_directory}, a git apply error will occur, but go to  ${pytorch_directory} and apply intel patch in ${intel_pytorch_extension_directory} will success.

Reply:
thanks for your comments.